src/App.js

```diff
@@ -2,10 +2,31 @@ import React from "react";
 import "./App.css";
 import Button from "./components/Button.jsx";
 import Footer from "./components/Footer.jsx";
+import Header from './components/Header.jsx';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import logo from './logo2.png';
+
 
 function App() {
```

Comment:
Make 'App' a  class based component and try to mangage only routing here. Make a layout component (higher order component ) which can be use to bind header, sidedrawer , footer and body and then enclose all the routes here inside the layout component.